es/index.mjs

```diff
@@ -7,12 +7,17 @@ export const namespace = 'axios-retry';
  * @return {boolean}
  */
 export function isNetworkError(error) {
-  return (
-    !error.response &&
-    Boolean(error.code) && // Prevents retrying cancelled requests
-    error.code !== 'ECONNABORTED' && // Prevents retrying timed out requests
-    isRetryAllowed(error)
-  ); // Prevents retrying unsafe errors
+	const CODE_EXCLUDE_LIST = [
+		'ERR_CANCELED',
+		'ECONNABORTED'
+	];
+
+	return (
+		!error.response &&
+		Boolean(error.code) && // Prevents retrying cancelled requests
```

Comment:
are they not excluded already with this condition?

Reply:
Nope! Cancel error did retry

Reply:
sorry man, completely forgot about this
you can use the retryCondition to customise this but I think it makes sense by default to not retry on this error (which is triggered for example when using an abort controller)
merging it

Reply:
published as 3.8.0
thanks a lot @GaryPEGEOT